go.mod

```diff
@@ -3,25 +3,28 @@ module github.com/operator-framework/operator-registry
 require (
 	github.com/antihax/optional v0.0.0-20180407024304-ca021399b1a6
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-bindata/go-bindata v3.1.2+incompatible // indirect
+	github.com/golang-migrate/migrate/v4 v4.6.2
 	github.com/golang/mock v1.2.0
 	github.com/golang/protobuf v1.3.1
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/grpc-ecosystem/grpc-health-probe v0.2.1-0.20181220223928-2bf0a5b182db
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/mattn/go-sqlite3 v1.10.0
+	github.com/operator-framework/operator-lifecycle-manager v3.11.0+incompatible // indirect
```

Comment:
Where did this come from? We recently did some work to remove the dependency on OLM. 

(also this is picking up the 3.11 version which is like 0.7.0 I think)